### DISCUSSION TITLE
宝塔终端使用pip install nb-cli指令报错

### DISCUSSION THREAD

**Discussion by GFCDIAO** [nonebot/discussions#244] [问答] [answered]
![image](https://user-images.githubusercontent.com/118088275/202133881-77cd9df6-a26b-40c4-8d84-e0510e85a2a0.png)
如题，在宝塔终端venv虚拟环境下pip install nb-cli出现以上错误，请问是否是python版本问题？该如何解决
云端服务器用的是腾讯云
![image](https://user-images.githubusercontent.com/118088275/202138310-6ffd82c7-c95d-418d-bbc7-7efaa2043dbf.png)
这里显示的是阿里源

**Comment by yanyongyu** [comment, marked as answer]
python版本太低，pip也没更新

**Reply by GFCDIAO** [reply]
大佬请问宝塔的python版本该如何更新，我用过了它的python管理器，然后升级到了3.10版本，但是终端里面的python -V指令还是python 3.7.9

**Reply by yanyongyu** [reply]
没用过宝塔的管理器（

**Reply by GFCDIAO** [reply]
哦哦好的，我去官网客服问问，非常感谢🙏
